src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -161,7 +162,14 @@ const ColonyHome = ({ match, location }: Props) => {
               path={COLONY_EXTENSIONS_ROUTE}
               component={() => <>Extensions</>}
             />
-            <Route path={COLONY_HOME_ROUTE} component={() => <>Actions</>} />
+            <Route
+              path={COLONY_HOME_ROUTE}
+              component={() => (
+                <div className={styles.actions}>
+                  <ColonyActions />
```

Comment:
Not sure about this wrapper div with styles here? Maybe we should put it inside ColonyActions?